platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureFragment.kt

```diff
@@ -242,8 +258,8 @@ open class FeatureFragment : Fragment() {
             }
         }
         startActivityForResult(intent, PICK_FILE_REQUEST_CODE)
-        var url: String = ""
-        var name: String = ""
+        var url = ""
```

Comment:
Again, what's the reason for this? Fixing linting warnings? Also, you're keeping the typing for size. Why is that not the same case?

Reply:
yep. the type here is obsolete as the compiler knows it's a string if there's a value initialization.